WORKSPACE

```diff
@@ -100,6 +100,16 @@ new_local_repository(
     workspace_file = "//third_party/rules_python:rules_python.WORKSPACE",
 )
 
+# This is the real rules_python.  It's needed for a specific bug reproduction case.
+http_archive(
+    name = "real_rules_python",
+    sha256 = "cdf6b84084aad8f10bf20b46b77cb48d83c319ebe6458a18e9d2cebf57807cdd",
+    strip_prefix = "rules_python-0.8.1",
+    urls = [
+        "https://github.com/bazelbuild/rules_python/archive/refs/tags/0.8.1.tar.gz",
+    ],
+)
+
```

Comment:
If this is a Bazel issue, shouldn't it be possible to test it without bringing in a new dependency?

We are trying to keep the list of dependencies minimal. Additional one just for a single test is not an option.

Reply:
This is a bazel issue -- the bug is in python_stub_template.txt.

This is a bazel issue.  It can be exposed without the rules_python package, but that would require recreating about 100 lines of python code from rules_python.  It's probably best to test this functionality using the version of rules_python that our customers are using.